Gems/DiffuseProbeGrid/Code/Source/Render/DiffuseProbeGrid.h

```diff
@@ -55,6 +55,116 @@ namespace AZ
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeNormalBiasNameIndex = "m_probeGrid.probeNormalBias";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeViewBiasNameIndex = "m_probeGrid.probeViewBias";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeDistanceExponentNameIndex = "m_probeGrid.probeDistanceExponent";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeIrradianceThresholdNameIndex = "m_probeGrid.probeIrradianceThreshold";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeBrightnessThresholdNameIndex = "m_probeGrid.probeBrightnessThreshold";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeIrradianceEncodingGammaNameIndex = "m_probeGrid.probeIrradianceEncodingGamma";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeRandomRayBackfaceThresholdNameIndex = "m_probeGrid.probeRandomRayBackfaceThreshold";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeFixedRayBackfaceThresholdNameIndex = "m_probeGrid.probeFixedRayBackfaceThreshold";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeMinFrontfaceDistanceNameIndex = "m_probeGrid.probeMinFrontfaceDistance";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeScrollOffsetsNameIndex = "m_probeGrid.probeScrollOffsets";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeRayDataFormatNameIndex = "m_probeGrid.probeRayDataFormat";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeIrradianceFormatNameIndex = "m_probeGrid.probeIrradianceFormat";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeRelocationEnabledNameIndex = "m_probeGrid.probeRelocationEnabled";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeClassificationEnabledNameIndex = "m_probeGrid.probeClassificationEnabled";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeScrollClear0NameIndex = "m_probeGrid.probeScrollClear[0]";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeScrollClear1NameIndex = "m_probeGrid.probeScrollClear[1]";
+            RHI::ShaderInputNameIndex m_prepareSrgProbeGridProbeScrollClear2NameIndex = "m_probeGrid.probeScrollClear[2]";
+
+            RHI::ShaderInputNameIndex m_rayTraceSrgGridDataNameIndex = "m_gridData";
+            RHI::ShaderInputNameIndex m_rayTraceSrgProbeRayTraceNameIndex = "m_probeRayTrace";
+            RHI::ShaderInputNameIndex m_rayTraceSrgProbeIrradianceNameIndex = "m_probeIrradiance";
+            RHI::ShaderInputNameIndex m_rayTraceSrgProbeDistanceNameIndex = "m_probeDistance";
+            RHI::ShaderInputNameIndex m_rayTraceSrgProbeDataNameIndex = "m_probeData";
+            RHI::ShaderInputNameIndex m_rayTraceSrgAmbientMultiplierNameIndex = "m_ambientMultiplier";
+            RHI::ShaderInputNameIndex m_rayTraceSrgGiShadowsNameIndex = "m_giShadows";
```

Comment:
It seems all of these shader input indexes are not used anywhere. Do you need to add them here?

Reply:
They are used in DiffuseProbeGrid.cpp, which is included in the PR but considered a "Large Diff".  It is collapsed by default when you look at the files changes in the PR.